ion-schema-tests-runner/src/generator.rs

```diff
@@ -169,6 +175,30 @@ fn generate_test_cases_for_file(ctx: Context) -> TokenStream {
     join_token_streams(test_case_tokens)
 }
 
+/// find ISL version from schema content
+fn find_isl_version(schema_content: &[Element]) -> IonSchemaLanguageVersion {
```

Comment:
The logic of this function is not correct. See my comments on the other copy of this function.

Reply:
Changed with a correct implementation.